### PR TITLE
CDN install instructions

### PR DESCRIPTION
**Pull request by xjensen**
Per #324, updates component documentation with CDN installation instructions.

Also includes #923 by adding `dist` folders for all CSS-only components. Link-icon also gets a `dist` folder, and it's CSS is now inserted via JS to bring it into consistency with the other JavaScript components.

### CODE REVIEW

**Comment by xjensen** [comment]
Thus far, I've only updated the **accordion** documentation. If this looks good, let me know and I'll go through and do the rest.

**Review by aaronhans — APPROVED** [review]
good

**Comment by zakiya** [comment]
See my comment at https://github.com/cagov/design-system/issues/324

**Comment by xjensen** [comment]
I've added CDN installation instructions for a majority of the components.

Some have been excluded due to inconsistent file paths for CSS/JS, which has led to broken CDN uploads. #923 describes this problem. I think we'll need to work through that issue to fully finish this PR.

**Comment by xjensen** [comment]
#923 has been added to this PR.

* All components now have a `dist` folder, including CSS-only components.
* Major version numbers have been bumped in affected components.
* Link-icon gets a `dist` folder too. It's been modified to insert CSS via JS like the other JavaScript components.
* The CDN publisher has been altered to support these changes. It will now publish CSS to the `dist` folders.

I did not add CDN installation instructions to **base-css** and **combined-css**. We need to do a lot more with those READMEs to make them useful, so I'm punting.

I also skipped **icons**. I suspect there may be a larger discussion worth having about how to make that available via CDN.

Finally, I feel we should remove the component `dist` folders from the Git repo. They are build artifacts. We would still include them in the NPM packages, of course. But that's a project for another day.

**Comment by zakiya** [comment]
> All components now have a `dist` folder, including CSS-only components.

This sparks joy. ✨

**Comment by xjensen** [comment]
@aaronhans @zakiya I'd like to get a fresh review on this PR if y'all don't mind. It's changed quite a bit since the previous approval, including major version bumps to half the components.

File: docs/src/publish/cdn-deploy.js [hunk 1]
```diff
@@ -49,16 +48,11 @@ const getFilePaths = (dir) => {
...
-addToFilePaths(cssToUpload);
 
 // upload to S3
 const uploadToS3 = (dir, filePath) =>
   new Promise((resolve, reject) => {
-    let key = keyPrefix + filePath.split(`${dir}/`)[1];
-    if (filePath.indexOf('.css') > -1) {
-      key = cssKey;
-    }
+    const key = keyPrefix + filePath.split(`${dir}/`)[1];
```
Comment (aaronhans) [inline docs/src/publish/cdn-deploy.js:55]: nice

**Review by aaronhans — APPROVED** [review]
This is great @xjensen! Love all the major version bumps and standardization we have now

**Comment by xjensen** [comment]
I'm going to merge into release branch and publish to NPM. The major version bumps should keep all our sites safe.

**Pull request merged**